pkg/config/resources/terraform/provider.go

```diff
@@ -321,7 +321,12 @@ func (p *TerraformProvider) terraformApply(containerid string) error {
     -auto-approve`
 
 	// add the tf vars flag if we have a file
-	script = script + tfvarFlag
+	// otherwise add a newline to separate the apply and output commands
+	if tfvarFlag != "" {
+		script = script + tfvarFlag
+	} else {
+		script = script + "\n"
+	}
```

Comment:
Shouldn't we just always add a `;` to separate the commands?

Reply:
I'd rather rework how we build up the commands, once the resources are pulled into the sdk

Reply:
Sure, could even consider using https://pkg.go.dev/github.com/hashicorp/terraform-exec instead.